ipywidgets/widgets/domwidget.py

```diff
@@ -11,11 +11,20 @@
 
 
 class DOMWidget(Widget):
-    """Widget that can be inserted into the DOM"""
+    """Widget that can be inserted into the DOM
+
+    Parameters
+    ----------
+    tooltip: str
+       tooltip caption
+    layout: InstanceDict(Layout)
+       widget layout
+    """
 
     _model_name = Unicode('DOMWidgetModel').tag(sync=True)
     _dom_classes = TypedTuple(trait=Unicode(), help="CSS classes applied to widget DOM element").tag(sync=True)
     tabbable = Bool(help="Is widget tabbable?", allow_none=True, default_value=None).tag(sync=True)
+    tooltip = Unicode(None, allow_none=True, help="A tooltip caption.").tag(sync=True)
```

Comment:
I think we can leave the default as empty string (the default `default` value), and not specify `allow_none=True`.